akka-http/src/main/scala/akka/http/scaladsl/server/Route.scala

```diff
@@ -22,6 +22,8 @@ object Route {
 
   /**
    * "Seals" a route by wrapping it with exception handling and rejection conversion.
+   *
+   * A "sealed" route accepts all requests making any routes after it unreachable at runtime.
```

Comment:
> `seal` converts all rejections and exceptions in the wrapped route to responses.  Any any routes concatenated after it will be unreachable at runtime.

^?

I'm afraid I am not yet well enough versed in Akka HTTP to know if  the "etc" in your response confers something I don't even know about.

For me the important part was mainly the "causing any routes concatenated after it to be unreachable at runtime." (slightly modified from the above) since that seems to be the behavior user would see when reasoning about what it does.